docs/pywbemcligeneraloptions.rst

```diff
@@ -38,25 +36,25 @@ information on each command-group arguments and options):
   A new server (``myserver``)may be defined and saved in the connection file with a
   name is defined as follows::
 
-    $ pywbemcli -s http://localhost --name myserver --user user --password password connection save
+    $ pywbemcli add -s http://localhost --name myserver --user user --password password
 
-  To use an existing server named ``myserver`` in the defined connections:
+  To use an existing server named ``myserver`` in the defined connections::
 
     $ pywbemcli --name myserver class get CIM_ManagedElement
 
   See :ref:`Connection command-group` for more information on managing
   connections.
 
-* **--default_namespace/-d** - Default :term:`CIM namespace` to use in the target
-   WBEM server if no namespace is defined in a command. If not defined the
-   pywbemcli default is ``root/cimv2``.  This is the namespace used on all
-   server operation requests unless a specific namespace is defined by:
+* **--default-namespace/-d** - Default :term:`CIM namespace` to use in the target
+  WBEM server if no namespace is defined in a command. If not defined the
+  pywbemcli default is ``root/cimv2``.  This is the namespace used on all
+  server operation requests unless a specific namespace is defined by:
 
-   * In the interactive mode prepending the command-group name with the
-     ``--namespace`` option.
-   * Using the ``--namespace`` or ``-n`` command option to define a namespace
-     on subcommands that specify this option.
-   * Executing a command that looks in multiple namespaces (ex. ``class find``).
+  * In the interactive mode prepending the command-group name with the
+    ``--namespace`` option.
+  * Using the ``--namespace`` or ``-n`` command option to define a namespace
+    on subcommands that specify this option.
+  * Executing a command that looks in multiple namespaces (ex. ``class find``).
 * **--user/-u** - Username for the WBEM server if a user name is required to
   authenticate the client.
 * **--password/-p** - Password for the WBEM server. This option is normally
```

Comment:
Three lines down, use single backquotes in link ":ref:\`\`Avoiding password prompts\`\`"

Reply:
DONE